mm2src/mm2_main/src/docker_tests.rs

```diff
@@ -442,6 +442,34 @@ mod docker_tests {
         assert_eq!(FoundSwapTxSpend::Refunded(refund_tx), found);
     }
 
+    #[test]
+    fn test_for_non_existent_tx_hex_utxo() {
```

Comment:
If you can add another test for electrums, please do :)

Reply:
Yes I added another unit test for electrums in `utxo_tests.rs`